src/shared/modules/dbMeta/dbMetaDuck.ts

```diff
@@ -531,6 +534,7 @@ export const dbMetaEpic = (some$: any, store: any) =>
     .ofType(UPDATE_CONNECTION_STATE)
     .filter((s: any) => s.state === CONNECTED_STATE)
     .merge(some$.ofType(CONNECTION_SUCCESS))
+    .merge(some$.ofType(SWITCH_CONNECTION_SUCCESS))
```

Comment:
Is this really needed?
I _think_ it's already triggered by the lines right above by the connection state going into `CONNECTED_STATE`.

Reply:
From our conversation earlier I thought they were separate & distinct events. I guess the best way to clear up that question is by performing the actions in desktop that trigger it.